src/styles/main.css

```diff
@@ -0,0 +1,188 @@
+body {
+  margin: 0;
+  font-family: Arial, Baghdad, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: 720px;
+  gap: 1px;
+  margin: 50px auto;
+}
+.calendar--month-length--28 .day:nth-child(n+29) {
+  display: none;
+}
+.calendar--month-length--29 .day:nth-child(n+30) {
+  display: none;
+}
+.calendar--month-length--30 .day:nth-child(n+31) {
+  display: none;
+}
+.calendar--month-length--31 .day:nth-child(n+32) {
+  display: none;
+}
+.calendar__start-day--mon > :first-child {
```

Comment:
The start day modifier does not account for the 1px gap between days. Each day should be offset by 101px, not 100px.